src/dom.js

```diff
@@ -270,3 +271,24 @@ export function childElementByTag(parent, tagName) {
     return el.tagName == tagName;
   });
 }
+
+
+/**
+ * Returns element attributes that pass the callback test as params.
+ * @param {!Element} element
+ * @param {function(!Element):string|null} parseAttributeCallback
+ * @return {Object}
+ */
+export function getAttributesAsParams(element, parseAttributeCallback) {
```

Comment:
Yes we can. I thought we would keep this a bit general to allow other manipulation to params but I guess I don't see any use case for them right now. Is `getDataParamsFromAttributes` a good name?